server/views/pages/index.njk

```diff
@@ -1,24 +1,30 @@
 {% extends "../partials/layout.njk" %}
-{% from "govuk/components/table/macro.njk" import govukTable %}
 
 {% set pageId = 'index-page' %}
-{% set pageTitle = applicationName + " - Home" %}
-{% set mainClasses = "app-container govuk-body" %}
+{% set pageTitle = "Manage learning and work progress" %}
 
 {% block beforeContent %}
-  {% include "../partials/breadCrumb.njk" %}
+{{ govukBreadcrumbs({
```

Comment:
Why is this breadcrumbs different?

Reply:
Best practice is that the breadcrumb does not show the current page, so this overrides the shared Breadcrumb partial to remove the current page.